forte/casscf/casscf_orb_grad.cc

```diff
@@ -694,7 +718,7 @@ bool CASSCF_ORB_GRAD::update_orbitals(psi::SharedVector x) {
     dR->subtract(R_);
 
     // incoming x consistent with R_, no need to update orbitals
-    if (dR->absmax() < 1.0e-12)
+    if (dR->absmax() < 1.0e-15)
```

Comment:
This value is very suspicious. Can you at least make it a variable (maybe class variable) and give it a name?

Reply:
This is for numerical zero. Ideally, we could add an global Forte option for numerical zero.